_scalingo_api/log_drains.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Log drains
+layout: default
+---
+
+# Log drains
+
+Log from an application can be redirect to a management service.
+You can add multiple log drains from a single application.
```

Comment:
In this sentence you talk about "log drains" but you never explained what a "log drain" is. You should add a sentence to explain.

```suggestion
You can add multiple log drains to a single application.
```